packages/wallet-service/tests/txProcessor.test.ts

```diff
@@ -1,441 +1,19 @@
-    await expect(checkAddressTable(mysql, 4, decoded.address, null, null, 1)).resolves.toBe(true);
-    // there are 4 different addresses with some balance
-    await expect(checkAddressBalanceTable(mysql, 4, decoded.address, token, value, 0, null, 1)).resolves.toBe(true);
-    await expect(checkAddressTxHistoryTable(mysql, 6, decoded.address, tx.tx_id, token, value, tx.timestamp)).resolves.toBe(true);
-  }
-  for (const input of tx.inputs) {
-    const { decoded, token, value } = input;
-    // the input will have a negative amount in the address_tx_history table
-    await expect(checkAddressTxHistoryTable(mysql, 6, decoded.address, tx.tx_id, token, (-1) * value, tx.timestamp)).resolves.toBe(true);
-  }
-  // address1 balance has decreased
-  await expect(checkAddressBalanceTable(mysql, 4, 'address1', '00', blockReward, 0, null, 3)).resolves.toBe(true);
-  // address2 balance is still locked
-  await expect(checkAddressBalanceTable(mysql, 4, 'address2', '00', 0, blockReward, null, 1)).resolves.toBe(true);
-});
-
-test.skip('txProcessor should be able to re-process txs that were voided in the past', async () => {
-  expect.hasAssertions();
-
-  const walletId = 'walletId';
-  const txId = 'txId1';
-  const address = 'address1';
-  const tokenId = '00';
-
-  await addToWalletTable(mysql, [{
-    id: walletId,
-    xpubkey: XPUBKEY,
-    authXpubkey: AUTH_XPUBKEY,
-    status: 'ready',
-    maxGap: 10,
-    createdAt: 1,
-    readyAt: 2,
-  }]);
-
-  await addToAddressTable(mysql, [
-    { address, index: 0, walletId, transactions: 1 },
-  ]);
-
-  // receive a block
-  const evt = JSON.parse(JSON.stringify(eventTemplate));
-  const block = evt.Records[0].body;
-  const blockUtxo = createOutput(0, blockReward, address);
-  block.version = 0;
-  block.tx_id = txId;
-  block.height = 1;
-  block.inputs = [];
-  block.outputs = [blockUtxo];
-
-  await txProcessor.onNewTxEvent(evt);
-
-  const logger = createDefaultLogger();
-
-  // void it
-  const transaction = await fetchTx(mysql, txId);
-  await commons.handleVoided(mysql, logger, transaction);
-
-  // call it again with the same tx
-  await txProcessor.onNewTxEvent(evt);
-
-  expect(await getTxOutput(mysql, txId, 0, false)).toStrictEqual({
-    txId,
-    index: 0,
-    tokenId,
-    address,
-    value: blockReward,
-    authorities: 0,
-    timelock: null,
-    heightlock: 2,
-    locked: true,
-    txProposalId: null,
-    txProposalIndex: null,
-    spentBy: null,
-  });
-
-  expect(await getWalletTxHistory(mysql, walletId, tokenId, 0, 10)).toStrictEqual([
-    {
-      txId: 'txId1',
-      timestamp: expect.anything(),
-      voided: 0,
-      balance: 6400,
-      version: 0,
-    },
-  ]);
-
-  expect(await checkAddressTxHistoryTable(
-    mysql,
-    1,
-    address,
-    txId,
-    tokenId,
-    blockReward,
-    block.timestamp,
-  )).toStrictEqual(true);
-});
-
-test.skip('txProcessor should ignore NFT outputs', async () => {
```

Comment:
We are still ignoring NFT outputs, but have no tests for this case, we should add it